.github/issue_template.md

```diff
@@ -0,0 +1,11 @@
+## Expected Behavior
```

Comment:
It's pretty conditional, sadly.

Maybe something like "Why would you like this feature?"